src/components/map/MapService.js

```diff
@@ -1063,14 +1013,8 @@ goog.require('ga_urlutils_service');
         };
 
         this.getConfig3d = function(config) {
-          while (config.config3d) {
-            var config3d = layers[config.config3d];
-
-            config = angular.extend({}, config, config3d);
-            if (!config3d.config3d) {
-              // avoid infinite loop
-              config.config3d = undefined;
-            }
+          if (config.config3d) {
+            return layers[config.config3d];
```

Comment:
so how did you do when a layer in 3d has a different timestamp or a different format or a different attribution?